lib/plugins/bidirectionalData.js

```diff
@@ -26,7 +26,7 @@
 const async = require('async');
```

Comment:
An entry in CHANGES_NEXT_RELEASE file about the changes in this PR should be added.

Reply:
Fixed e0220ae